apps/prairielearn/src/ee/pages/instructorAiGenerateQuestion/instructorAiGenerateQuestion.ts

```diff
@@ -193,6 +194,21 @@ router.post(
       );
 
       res.redirect(res.locals.urlPrefix + '/question/' + qid + '/settings');
+    } else if (req.body.__action === 'delete_drafts') {
+      const questions = await queryRows(
+        sql.select_all_drafts,
+        { course_id: res.locals.course.id.toString() },
+        QuestionSchema,
+      );
+
+      for (const question of questions) {
+        const locals = res.locals;
+        locals['question'] = question;
+        const editor = new QuestionDeleteEditor({ locals });
```

Comment:
Is there a better way to do this than making a new job for each question to be deleted/sync to be done each time? I see that prepareServerJob has some checks that are protected.

Reply:
Would just using an editor to batch-remove *all* of the `/__drafts__/` files be preferred?

Reply:
Discussed offline, we're going to ignore draft removal for the time being in the interest of shipping something more quickly.